Sort/MergeIntervals.swift

```diff
@@ -1,44 +1,29 @@
 /**
  * Question Link: https://leetcode.com/problems/merge-intervals/
- * Primary idea: Sort the original intervals and then append them one by one
+ * Primary idea: Sort intervals by leading integer in ascending order and append based on their leading and trailing integers.
  * Time Complexity: O(nlogn), Space Complexity: O(n)
- *
- * Definition for an interval.
- * public class Interval {
- *   public var start: Int
- *   public var end: Int
- *   public init(_ start: Int, _ end: Int) {
- *     self.start = start
- *     self.end = end
- *   }
- * }
  */
 
 class MergeIntervals {
-    func merge(intervals: [Interval]) -> [Interval] {
-        var result = [Interval]()
-        
-        let intervals = intervals.sorted {
-            if $0.start != $1.start {
-                return $0.start < $1.start
+    func merge(_ intervals: [[Int]]) -> [[Int]] {
```

Comment:
Your solution works but it is almost the same logic as the original one. Since it does not help improve the runtime complexity or simplify the code, I will not merge it.